includes/MediaWiki/Api/EditEndpoint.php

```diff
@@ -34,16 +37,21 @@ class EditEndpoint extends SimpleHandler {
 	/** @var SimplePutStrategy */
 	private $simplePutStrategy;
 
+	/** @var User */
+	private $user;
```

Comment:
Is this supposed to be a `User` or a `UserIdentity`? It’s `UserIdentity` in the constructor but `User` in the phpdoc comment.